platform/core/Makefile

```diff
@@ -15,19 +15,19 @@ ios_targets := iPhone simulator
 tools_dir := tools
 android_core := PolyPodCoreAndroid/core/src/main/
 android_version := 24
-# Rust still uses old-ish r22 version of NDK.
-# For reference https://github.com/rust-lang/rust/pull/85806
-ndk_version := r22b
 google_repo := https://dl.google.com/android/repository
-ndk_lib := android-ndk-$(ndk_version)
-ndk=$(tools_dir)/NDK_$(ndk_version)
-ndk_archive = $(tools_dir)/NDK.zip
-export ANDROID_NDK_HOME := $(ndk)/$(ndk_lib)
 android_triples := aarch64-linux-android armv7-linux-androideabi x86_64-linux-android i686-linux-android
 android_binaries := $(patsubst %, target/%, $(android_triples))
 android_architectures := arm64-v8a armeabi-v7a x86_64 x86
 jni_libs_copy_path := $(android_core)/jniLibs
 android_core_libraries := $(patsubst %, $(jni_libs_copy_path)/%, $(android_architectures))
+# Android NDK fix
+# This is required due to Rust still using lgcc for NDK,
+# which was made osbsolete starting with NDK r23 in favor of lunwind.
+# The solution is to create a libgcc.a file in every clang architecture, which will link to -ilunwind.
+clang_path = ${ANDROID_NDK_HOME}/*/toolchains/llvm/prebuilt/*/lib64/clang/*/lib/*
+ndk_archs = aarch64 arm i386 x86_64
```

Comment:
Why do we still need this? I would say i386 is no longer used?

Reply:
This is meant to to be consistent with clang supported architectures.

Reply:
Right, but do *we* need it?

Reply:
TBH, I'm not totally sure about `aarch64` either. We need `arm` for iOS and I guess some Android, `x86_64` for, well, not sure either. Point is, we're building these binaries for our supported architectures, so shouldn't we cross-compile only for these? Do we need to bundle multi-arch binaries in our apps? In that case, which arch should we support?

Reply:
x86_64 is needed to be able to run on simulators.

Reply:
OK, gotcha. `arm` and `x86_64`. What about the others?

Reply:
aarch64 - > targets arm64 architectures.
Probably i386 is not needed, but point is - I am fixing NDK to work properly with rust for any architecture.